GitExtUtils/GitUI/Theming/AppColorDefaults.cs

```diff
@@ -15,8 +15,8 @@ public static class AppColorDefaults
                 { AppColor.HighlightAllOccurences, Color.LightYellow },
                 { AppColor.Tag, Color.DarkBlue },
                 { AppColor.Graph, Color.DarkRed },
-                { AppColor.Branch, Color.Green },
-                { AppColor.RemoteBranch, Color.DarkRed },
+                { AppColor.Branch, Color.FromArgb(34, 166, 57) },
+                { AppColor.RemoteBranch, Color.FromArgb(217, 57, 44) },
```

Comment:
the change needs to be repeated in `invariant.css`

Reply:
tests\UnitTests\GitUI.Tests\Theming\AppColorDefaultsTests.cs

There should have been a test to force this, but apparently I lost it somewhere.
```
        [Test]
        public void Invariant_theme_colors_match_AppColorDefaults()
        {
            var themePathProvider = new ThemePathProvider();
            var themeLoader = new ThemeLoader(new ThemeCssUrlResolver(themePathProvider), new ThemeFileReader());
            var repository = new ThemeRepository(new ThemePersistence(themeLoader), themePathProvider);
            var invariantTheme = repository.GetInvariantTheme();
            foreach (AppColor name in Enum.GetValues(typeof(AppColor)))
            {
                Color value = invariantTheme.GetColor(name);
                var defaultValue = AppColorDefaults.GetBy(name);
                value.ToArgb().Should().Be(defaultValue.ToArgb());
            }
        }
```

Reply:
#8901

Reply:
`invariant.css` has not been adapted. But the test passes. 🤔

Reply:
This is new test. This is fixed.